frontend/express/app.js

```diff
@@ -1177,10 +1179,15 @@ app.get(countlyConfig.path + '/api-key', function(req, res, next) {
 });
 
 app.get(countlyConfig.path + '/sdks.js', function(req, res) {
-    var options = {uri: "http://code.count.ly/js/sdks.js", method: "GET", timeout: 4E3};
-    request(options, function(a, c, b) {
-        res.set('Content-type', 'application/javascript').status(200).send(b);
-    });
+    if (!plugins.getConfig("api").offline_mode) {
+        var options = {uri: "http://code.count.ly/js/sdks.js", method: "GET", timeout: 4E3};
+        request(options, function(a, c, b) {
+            res.set('Content-type', 'application/javascript').status(200).send(b);
+        });
+    }
+    else {
+        console.log("this request cannot be made in offline mode.");
```

Comment:
can we reply with none 200 HTTP error code here, so browser would not wait for timeout